DeAceroDLabs/marketplace-frontend
# [DMP-610] Display possible values for every product property

## What is done

[[DMP-610] Despliega posibles valores para cada propiedad](https://deacero.atlassian.net/browse/DMP-610)

## How is done

1. Added a new property to the Button component to check if it's rounded or squared.
2. Added a new hover effect for secondary buttons.

## How to test

1. Run the project via `npm start`
3. Select any product from the products section.
4. Click on any arrow in the properties section to display the possible values.
5. You should see the possible values within the new buttons.
6. Hover over the buttons to see the hover effect.
